File: JAVA/5.Switch & ifelse/p6.java
import java.util.*;

class Spell{

        public static void main(String[] args){

                Scanner scan=new Scanner(System.in);
                System.out.println("Enter a number:");
                int ch=scan.nextInt();
                switch(ch){

                        case 1:
                                System.out.println("One");
                                break;
                        case 2:
                                System.out.println("Two");
                                break;
                        case 3:
                                System.out.println("Three");
                                break;
                        case 4:
                                System.out.println("Four");
                                break;
                        case 5:
                                System.out.println("Five");
                                break;
                        case 0:
                                System.out.println("Zero");
                                break;
                        default:
                                System.out.println("Invalid number");
                }
        }
}

File: Flutter/Assignment/Conditional Statement/p6.dart
import "Dart:io";
void main(){
	int? x;
	print("Enter a number:");
	x=int.parse(stdin.readLineSync()!);
	switch(x){
		case 0:
			print("Zero");
			break;
		case 1:
			print("One");
			break;
		case 2:
			print("Two");
			break;
		case 3:
			print("Three");
			break;
		case 4:
			print("Four");
			break;
		case 5:
			print("Five");
			break;
		default:
			print("$x is greater than 5");
	}
}
